fews_3di/utils.py

```diff
@@ -60,6 +61,7 @@ def __init__(self, settings_file: Path):
         self.settings_file = settings_file
         setattr(self, "lizard_results_scenario_name", "")
         setattr(self, "lizard_results_scenario_uuid", "")
+        setattr(self, "api_host", "https://api.3di.live/v3.0")
```

Comment:
To make it more explicit, I'd use a constant for this. Could you put something like this at the top of the file?

    DEFAULT_API_HOST = "https://api.3di.live/v3.0"

And then change this line to:

```suggestion
        setattr(self, "api_host", DEFAULT_API_HOST)
```